StepDefinitions/HotelsSD/HotelRemoveGusSD.js

```diff
@@ -0,0 +1,121 @@
+const { Given, Then } = require("@wdio/cucumber-framework");
+const { expect } = require("chai");
+const HotelHomePage = require("../../Pages/Hotels/HotelHomePage");
+
+const hotelhomepage = new HotelHomePage();
+
+Given(/^I am on hotels homepage$/, async function() {
+    await browser.url('/');
+    await browser.pause(1000);
+})
+
+Then(/^I click on Travelers$/, async function() {
+    await hotelhomepage.clickTravelersBox();
+    await browser.pause(2000);
+})
+
+Then(/^I select “Children” as 2$/, async function() {
+    await hotelhomepage.addChildren(2);
+    await browser.pause(2000)
+})
+
+Then(/^I verify Children-age dropdown are 2$/, async function() {
+    const isChild1DropDownDiplaying = await hotelhomepage.isChild1DropDownDiplayed();
+    const isChild2DropDownDiplaying = await hotelhomepage.isChild2DropDownDiplayed();
+
+    expect(isChild1DropDownDiplaying, 'Child 1 drop-down is NOT diplaying').to.be.true;
+    expect(isChild2DropDownDiplaying, 'Child 2 drop-down is NOT displaying').to.be.true;
+})
+
+Then(/^I verify Plus-button is enabled$/, async function() {
+    const isAddChildBttnEnable = await hotelhomepage.isAddChildBttnEnabled();
+    expect(isAddChildBttnEnable, 'Plus-button is NOT enabled').to.be.true;
+})
+
+Then(/^I verify minus-button is enabled$/, async function() {
+    const isRemoveChildBttnEnable = await hotelhomepage.isRemoveChildBttnEnabled();
+    expect(isRemoveChildBttnEnable, 'Minus-button is NOT enabled').to.be.true;
+})
+
+//2 children are already added so just need to add 4 more
+Then(/^I select “Children” as 6$/, async function() {
+    await hotelhomepage.addChildren(4);
+    await browser.pause(2000);
+})
+
+Then(/^I verify Children-age dropdown are 6$/, async function() {
+    const isChild1DropDownDiplaying = await hotelhomepage.isChild1DropDownDiplayed();
+    const isChild2DropDownDiplaying = await hotelhomepage.isChild2DropDownDiplayed();
+    const isChild3DropDownDiplaying = await hotelhomepage.isChild3DropDownDiplayed();
+    const isChild4DropDownDiplaying = await hotelhomepage.isChild4DropDownDiplayed();
+    const isChild5DropDownDiplaying = await hotelhomepage.isChild5DropDownDiplayed();
+    const isChild6DropDownDiplaying = await hotelhomepage.isChild6DropDownDiplayed();
+
+    expect(isChild1DropDownDiplaying, 'Child 1 drop-down is NOT diplaying').to.be.true;
+    expect(isChild2DropDownDiplaying, 'Child 2 drop-down is NOT displaying').to.be.true;
+    expect(isChild3DropDownDiplaying, 'Child 3 drop-down is NOT diplaying').to.be.true;
+    expect(isChild4DropDownDiplaying, 'Child 4 drop-down is NOT diplaying').to.be.true;
+    expect(isChild5DropDownDiplaying, 'Child 5 drop-down is NOT diplaying').to.be.true;
+    expect(isChild6DropDownDiplaying, 'Child 5 drop-down is NOT diplaying').to.be.true;
+    await browser.pause(2000);
```

Comment:
These looks like lot of work and not efficient. You can get count like below and verify the count.

        const allChildrenAgeDropdownArray = await $$(this.allChildrenAgeDropdownLocator);
        return allChildrenAgeDropdownArray.length;